config/200-clusterrole-namespaced.yaml

```diff
@@ -0,0 +1,10 @@
+kind: ClusterRole
```

Comment:
Can you add a license header (see other yaml files)

Reply:
Sure, I added it. Thank you.